.cicd/platforms/pinned/amazon_linux-2-pinned.dockerfile

```diff
@@ -13,8 +13,9 @@ RUN curl -s https://packagecloud.io/install/repositories/rabbitmq/erlang/script.
     yum install -y erlang
 RUN curl -s https://packagecloud.io/install/repositories/rabbitmq/rabbitmq-server/script.rpm.sh | bash && \
     yum install -y rabbitmq-server
-# upgrade pip installation
-RUN pip3 install --upgrade pip
+# upgrade pip installation. request and requests_unixsocket module
+RUN pip3 install --upgrade pip && \
+    pip3 install requests && pip3 install requests_unixsocket
```

Comment:
This can just be 
`pip3 install requests requests_unixsocket `

Reply:
Done.